### ISSUE TITLE
Create README file for each example including image/snapshot?

### ISSUE THREAD

**Issue by amirhouieh** [openrndr/openrndr-tutorials#1]
I think is good that if each example `README` contains an image/snapshop of the output of that example.
@edwinRNDR I can get on it, if you agree.

**Comment by edwinRNDR** [comment]
@amirhouieh That's good. Let's set some style guidelines first; including simple things like the common window size, should window chrome be included?

**Comment by amirhouieh** [comment]
@edwinRNDR good idea. Most of examples use the default window sizes (`640`, `480`), so I would say lets use this dimensions. 
I think we should include window chrome, since it would indicate openRNDR immediately, and it just feels right to me. 

Btw, we are missing `README` files as well. So here is a simple structure I propose:
```
Title of example
Short explanation about what this is (same as comments in the code)
[image]
```
